oracle-plugin/widgets/Oracle-batchsink.json

```diff
@@ -150,6 +150,18 @@
             "label": "Browse Database"
           }
         },
+        {
+          "widget-type": "select",
+          "label": "Transaction Isolation Level",
```

Comment:
Since, this property is included in `ConnectionConfig`, I would suggest to move it under `Connection` subheading below `Role` instead of `Basic`.

Reply:
so any configuration I add within the ConnectorConfig class would need to be applied to the connector section of a widget? still learning how these plugins and java classes mesh.

Reply:
> so any configuration I add within the ConnectorConfig class would need to be applied to the connector section of a widget? still learning how these plugins and java classes mesh.

It is not mandatory but generally the pattern followed in all plugins. These fields should only be shown when `UseConnection` set to `False`.

Reply:
got. this is now fixed